style.css

```diff
@@ -377,3 +374,237 @@ fieldset {
   right: 7rem;
   margin-top: -50px;
 }
+
+.nav-link {
+  display: none;
+}
+
+.ruby {
+  display: none;
+}
+
+.el {
+  display: initial;
+}
+
+.about-1 {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+@media screen and (min-width: 768px) {
+  * {
+    overflow-x: hidden;
+  }
+
+  header {
+    padding: 1rem 5rem;
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    background-color: white;
+  }
+
+  header .menu {
+    display: none;
+  }
+
+  .nav-link {
+    display: initial;
+  }
+
+  .nav-link ul {
+    display: flex;
+    align-items: center;
+  }
+
+  .nav-link li a {
+    font-size: 1.3rem;
+    padding-left: 5rem;
+    color: #344563;
+  }
+
+  .section-1 {
+    padding: 0 10rem 0 5rem;
+    width: 80vw;
```

Comment:
Great job with the project so far. However, setting a fixed width for this section is causing unusual behavior on larger screens. I have recorded [this video](https://www.loom.com/share/f5e07483e641494b84a8d37b17b2fa51) where I tried to explain exactly what the issue is. Kindly fix the issue and then submit for the re-review.